### ISSUE TITLE
eMotimo TB3 support

### ISSUE THREAD

**Issue by leandroprz** [timelapseplus/VIEW#43]
Hi,

Are there any plans on adding support for the TB3 Black to track the Sun/Moon?

Thank you

**Comment by elijahparker** [comment]
No, the TB3 only supports synchronizing with the VIEW, not full control, so
it can’t support motion features like tracking or key frames.
